python/ray/serve/_private/logging_utils.py

```diff
@@ -1,18 +1,60 @@
 import logging
 import os
 from typing import Optional
+import json
+import copy
 
 import ray
-from ray.serve._private.constants import DEBUG_LOG_ENV_VAR, SERVE_LOGGER_NAME
+from ray.serve._private.constants import (
+    DEBUG_LOG_ENV_VAR,
+    SERVE_LOGGER_NAME,
+    JSONIFY_LOG_MESSAGE,
+)
 
 
 LOG_FILE_FMT = "{component_name}_{component_id}.log"
-COMPONENT_LOG_FMT = (
-    "%(levelname)s %(asctime)s {component_name} {component_id} "  # noqa:E501
-)
 MESSAGE_FMT = "%(filename)s:%(lineno)d - %(message)s"
-REQUEST_ID_FMT = "%(request_id)s "
-ROUTE_FMT = "%(route)s "
+
+# Whether to jsonify the log message.
+FLAGS_JSONIFY_LOG_MESSAGE = os.environ.get(JSONIFY_LOG_MESSAGE, "0") == "1"
```

Comment:
would prefer to define this directly in `constants.py` instead of adding the level of indirection